app/src/main/java/org/mozilla/fenix/compose/ClickableSubstringLink.kt

```diff
@@ -45,7 +46,10 @@ fun ClickableSubstringLink(
         )
 
         addStyle(
-            SpanStyle(color = FirefoxTheme.colors.textAccent),
+            SpanStyle(
+                color = textColor,
```

Comment:
Should this only be for when we have a wallpaper? All our existing links use textAccent and it is what it is defined for by the design system. So, it would seem strange to me that we were to decide to have a single instance of a link just being the underline outside of private mode.

Reply:
See also my comment [here.](https://github.com/mozilla-mobile/fenix/pull/26853#discussion_r965089966).

For the moment, as we are using it only in one place, I did not want to overcomplicate this composable. But it can be refactored to give the caller the chance to choose between span styles for the linked text, or even receive a `SpanStyle` as an argument if we choose to do so in the future. 

As a side note, taking into consideration the need for UX consistency, for me at least, it would be desirable if all links in the app are represented the same way.

Reply:
To be clear, I am talking about shouldn't this still be using text accent when we don't have a wallpaper?

Reply:
The current change would mean the link would be using text primary + underline in the default (no wallpaper) scenario, whereas I think we should be using textAccent for the default and textPrimary (textColor) when there is a wallpaper, but this might be a UX question.

Reply:
When talking offline with @Kate-Galetski we did not make a differentiation between having a wallpaper or not, but this as being the new styling of the link in this section. 
Maybe @Kate-Galetski can confirm I understood correctly.

Reply:
Ok, let's bring this up in the #firefox-mobile-eng-ux slack. I believe you're going on PTO next week. So, I can follow up on any changes.

Reply:
From @Kate-Galetski 
Hi. I am working on the Section Titles color (aka Jump Back in, Recently Saved) and links color (aka Show all, Learn more) when wallpaper is on.
 I am proposing to:
 Using Light / Text / Primary (Hex Code: 15141A) for dark text.
 Using Light / Text /On Color Primary (Hex Code: FBFBFE) for light text
When wallpaper is off (no wallpaper) please let's default to already agreed color styles.